pkg/cli/predict.go

```diff
@@ -67,10 +72,79 @@ the prediction on that.`,
 	cmd.Flags().StringVarP(&outPath, "output", "o", "", "Output path")
 	cmd.Flags().StringArrayVarP(&envFlags, "env", "e", []string{}, "Environment variables, in the form name=value")
 	cmd.Flags().BoolVar(&useReplicateAPIToken, "use-replicate-token", false, "Pass REPLICATE_API_TOKEN from local environment into the model context")
+	cmd.Flags().StringVar(&inputJSON, "json", "", "Pass inputs as JSON object, read from file (@inputs.json) or via stdin (@-)")
```

Comment:
`-` (without the `@` prefix) is the usual arg for "read from stdin". It's fine to support both but `-` is the preferred one. If you haven't seen it [this](https://clig.dev/?trk=public_post-text#arguments-and-flags) is a lovely CLI ux guide written by Ben :)

Reply:
Yeah, but that's usually added at the end of the command as a positional argument right? I can support it though.

The `@-` is lifted from cURL.

```
curl http://example.com --json @-
```

https://curl.se/docs/manpage.html#--json